src/modules/CaaSMapper.spec.ts

```diff
@@ -94,10 +94,10 @@ describe('CaaSMapper', () => {
       const item = mapper.registerReferencedItem(refId, path, 'remoteId')
 
       expect(mapper._remoteReferences).toEqual({
-        remoteId: { [`${refId}.${remotes.remoteId.locale}`]: [path] },
+        remoteId: { [`${refId}.${remotes.someName.locale}`]: [path] },
```

Comment:
`${refId}.${remotes.someName.locale}` has two usages - why do we not create const with that string? It would make the code much more readable